sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/AppendBlobAsyncClient.java

```diff
@@ -367,4 +368,55 @@
                 return new SimpleResponse<>(rb, item);
             });
     }
+
+    /**
+     * Seals an append blob, making it read only. Any subsequent appends will fail.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.blob.specialized.AppendBlobAsyncClient.seal}
+     *
+     * @return A reactive response signalling completion.
+     */
+    public Mono<Void> seal() {
+        try {
+            return sealWithResponse(new AppendBlobSealOptions())
+                .flatMap(FluxUtil::toMono);
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Seals an append blob, making it read only. Any subsequent appends will fail.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.blob.specialized.AppendBlobAsyncClient.sealWithResponse#AppendBlobSealOptions}
+     *
+     * @param options {@link AppendBlobSealOptions}
+     * @return A reactive response signalling completion.
+     */
+    public Mono<Response<Void>> sealWithResponse(AppendBlobSealOptions options) {
+        try {
+            return withContext(context -> sealWithResponse(options, context));
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    Mono<Response<Void>> sealWithResponse(AppendBlobSealOptions options, Context context) {
+        options = (options == null) ? new AppendBlobSealOptions() : options;
```

Comment:
Sorry to keep asking this, but it looks like we decided to allow null for options when they don't have a required value?

Reply:
yeah I noticed that we do allow null for options when they dont have a required value. Im gonna create a PR after this making them all have this functionality - just didnt want to have this PR do 2 things at once

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/13339/